components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/UpdateAdvancedConfigurations.java

```diff
@@ -71,4 +78,14 @@ private void updateCertificate(Certificate certificate, ServiceProvider serviceP
             }
         }
     }
+
+
+    private void handleAdditionalSpProperties(List<AdditionalSpProperty> spAdditionalProperties) {
+
+        // Support is not there only for POST or PATCH yet, Only for GET
+        if (!CollectionUtils.isEmpty(spAdditionalProperties)) {
```

Comment:
This line seems to be confusing as per the comment line above. What is the behaviour here? block this operation for POST/PATCH or block when additional properties is empty?

Reply:
We do not support the **additional properties** in POST/PATCH where the swagger file will have that as a part of request body. Hence, we decided to respond with 501. @senthalan too approved of this approach.